modules/microshift-lvms-csi-snapshot-optional.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-lvms-csi-snapshot-optional_{context}"]
+= LVMS CSI provider and CSI snapshot deployments
```

Comment:
```suggestion
= Disabling LVMS CSI provider and CSI snapshot deployments
```
And I would add the task here. You could potentially make this its own assembly, too.

Reply:
made the change, retaining this as concept module for now.